homeassistant/components/plugwise/quality_scale.yaml

```diff
@@ -81,29 +71,29 @@ rules:
     comment: This integration does not have any reconfiguration steps (yet) investigate how/why
   dynamic-devices:
     status: todo
-    comment: Add missing logic to button for unloading and creation
+    comment: Investigate missing logic to button for unloading and creation
```

Comment:
Nothing is missing in the `button` platform, the "missing" logic is not needed because the single button is connected to the gateway devices, which is, in principle, always present and doesn't change.
Also, no other buttons are expected to be added. If it ever comes to that, then the logic can be changed, if needed.

I suggest changing the status to `done`.

Reply:
@joostlek do you concur?

Reply:
Oh I see, you only create a button if the device is the gateway. That makes sense :+1:

Reply:
Updated accordingly